Items.cs

```diff
@@ -1,418 +1,430 @@
+        public const int ChestIkanaGrotto = 176; 
+        public const int ChestPiratesFortressRedRupee1 = 177;
+        public const int ChestPiratesFortressRedRupee2 = 178;
+        public const int ChestInsidePiratesFortressTankRedRupee = 179;
+        public const int ChestInsidePiratesFortressGuardSilverRupee = 180;
+        public const int ChestInsidePiratesFortressHeartPieceRoomRedRupee = 181;
+        public const int ChestInsidePiratesFortressHeartPieceRoomBlueRupee = 182;
+        public const int ChestInsidePiratesFortressMazeRedRupee = 183;
+        public const int ChestPinacleRockRedRupee1 = 184;
+        public const int ChestPinacleRockRedRupee2 = 185;
+        public const int ChestBomberHideoutSilverRupee = 186;
+        public const int ChestTerminaGrottoBombchu = 187;
+        public const int ChestTerminaGrottoRedRupee = 188;
+        public const int ChestTerminaUnderwaterRedRupee = 189;
+        public const int ChestTerminaGrassRedRupee = 190;
+        public const int ChestTerminaStumpRedRupee = 191;
+        public const int ChestGreatBayCoastGrotto = 192; 
+        public const int ChestGreatBayCapeLedge1 = 193; 
+        public const int ChestGreatBayCapeLedge2 = 194; 
+        public const int ChestGreatBayCapeGrotto = 195; 
+        public const int ChestGreatBayCapeUnderwater = 196; 
+        public const int ChestPiratesFortressEntranceRedRupee1 = 197;
+        public const int ChestPiratesFortressEntranceRedRupee2 = 198;
+        public const int ChestPiratesFortressEntranceRedRupee3 = 199;
+        public const int ChestToSwampGrotto = 200; 
+        public const int ChestDogRacePurpleRupee = 201;
+        public const int ChestGraveyardGrotto = 202; 
+        public const int ChestSwampGrotto = 203; 
+        public const int ChestWoodfallBlueRupee = 204;
+        public const int ChestWoodfallRedRupee = 205;
+        public const int ChestWellRightPurpleRupee = 206;
+        public const int ChestWellLeftPurpleRupee = 207;
+        public const int ChestMountainVillage = 208; 
+        public const int ChestMountainVillageGrotto = 209; 
+        public const int ChestToIkanaRedRupee = 210;
+        public const int ChestToIkanaGrotto = 211; 
+        public const int ChestInvertedStoneTowerSilverRupee = 212;
+        public const int ChestInvertedStoneTowerBombchu10 = 213;
+        public const int ChestInvertedStoneTowerBean = 214;
+        public const int ChestToSnowheadGrotto = 215; 
+        public const int ChestToGoronVillageRedRupee = 216; 
+        public const int ChestSecretShrineHeartPiece = 217; 
+        public const int ChestSecretShrineDinoGrotto = 218; 
+        public const int ChestSecretShrineWizzGrotto = 219; 
+        public const int ChestSecretShrineWartGrotto = 220; 
+        public const int ChestSecretShrineGaroGrotto = 221; 
+        public const int ChestInnStaffRoomGrotto = 222; // + are these two grottos?
```

Comment:
These are not grottos. Inn Staff Room is the room where you get the couple's mask on the last night. There's a chest in there. Inn Guest Room is the room you can open with the Room Key.

Reply:
Okay, I was thinking the same thing - but for some reason they were named grottos originally.